plugins/recruit-resources/src/components/EditVacancy.svelte

```diff
@@ -33,14 +33,16 @@
   const dispatch = createEventDispatcher()
 
   const client = getClient()
+  let oldName: string = ''
```

Comment:
Why we need it here? ( I think it or query refresh is extra)

Reply:
Fixed.